settings/src/components/totp.js

```diff
@@ -197,28 +198,18 @@ function createQrCode( data ) {
  */
 function SetupForm( { handleEnable, verifyCode, setVerifyCode, qrCodeUrl, secretKey } ) {
 	const verifyCodeLength = 6;
-	const canSubmit        = qrCodeUrl && secretKey && verifyCode && verifyCode.length === verifyCodeLength;
+	const cleanVerifyCode  = verifyCode.replaceAll( /\s/g, '' );
```

Comment:
Should we remove everything except numbers?

Reply:
I would also consider not updating the value after `verifyCodeLength` is reached. 

It's really easy to add an extra number and slip by the "enabled" state on the button and since the placeholder is gone, I would only have the label above the input to inform my next steps... but it's not likely that I read that carefully.

Reply:
> Should we remove everything except numbers?

I worry that could confuse the user; they might not understand why what they type wasn't showing up, and think there's a bug or something. Letting them see the `pattern/title` warning seems like a better UX.

> not updating the value after verifyCodeLength is reached

That seems more intuitive, but maybe has a similar issue to above. I played around with automatically triggering the warning if the length is too much, but then you have to hide it again when they backspace, and that opens questions about if it should appear when the count is < `6`... It starts to feel like we're overcomplicating it.

I'm open to it if you see an elegant way to do it, though.

Reply:
We could maybe just remove the `canSubmit` bit entirely, so that users can always click it, and when they do they'll see the `title` warning?

Reply:
> > Should we remove everything except numbers?
> 
> I worry that could confuse the user; they might not understand why what they type wasn't showing up, and think there's a bug or something. Letting them see the `pattern/title` warning seems like a better UX.

👍 

> 
> > not updating the value after verifyCodeLength is reached
> 
> That seems more intuitive, but maybe has a similar issue to above. I played around with automatically triggering the warning if the length is too much, but then you have to hide it again when they backspace, and that opens questions about if it should appear when the count is < `6`... It starts to feel like we're overcomplicating it.
> 
> I'm open to it if you see an elegant way to do it, though.

Yeah, I don't think triggering the warning is necessary, it would introduce too much to take in. I guess this is why some applications have the separate inputs. Ie: `[ ] [ ] [ ]  [ ] [ ] [ ]`.

This is not a deal breaker.

Reply:
Yeah, it'll be much nicer when/if we can [just use CSS to add a space in the middle](https://github.com/WordPress/two-factor/pull/518#discussion_r1110126364).